cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -12,124 +12,249 @@ import "protoc-gen-openapiv2/options/annotations.proto";
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
 
-  // A namespace can be provided if the package repositories for a specific namespace
-  // are requested, when supported by the packaging format.
-  string namespace = 2;
+  // Available package reference
+  //
+  // A reference uniquely identifying the package.
+  AvailablePackageReference available_package_ref = 1;
+  
+  // Latest available package version
+  //
+  // The latest version available for this package. Often expected when viewing
+  // a summary of many available packages.
+  string latest_version = 2;
+  
+  // Available package Icon URL
+  //
+  // A url for an icon.
+  string icon_url = 3;
+  
+  // Available package display name
+  //
+  // A name as displayed to users
+  string display_name = 4;
+  
+  // Available package short description
+  //
+  // A short description of the app provided by the package
+  string short_description = 5;
+}
+
+// AvailablePackageDetail
+//
+// An AvailablePackageDetail provides additional details required when
+// inspecting an individual package.
+message AvailablePackageDetail {
+  // TODO: add example for API docs
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
 
-  // TODO: Add standard filters. Potentially filter by plugin.
+  // Available package reference
+  //
+  // A reference uniquely identifying the package.
+  AvailablePackageReference package_ref = 1;
+  
+  // Available package version
+  //
+  // TODO: add description
+  string version = 2;
+  
+  // Available package icon URL
+  //
+  // A url for an icon.
+  string icon_url = 3;
+  
+  // Available package display name
+  //
+  // A name as displayed to users
+  string display_name = 4;
+  
+  // Available package short description
+  //
+  // A short description of the app provided by the package
+  string short_description = 5;
+  
+  // Available package long description
+  //
+  // A long description of the package, such as a README formatted with Markdown.
+  string long_description = 6;
+  
+  // Available package default values
+  //
+  // An example of default values used during package templating that can serve
+  // as documentation or a starting point for user customization.
+  string default_values = 7;
+  
+  // Available package values schema
+  //
+  // An optional openapi/json schema that can be used to validate a user-provided values.
+  string values_schema = 8;
+  
+  // Available package maintainers
+  //
+  // List of Maintainer
+  repeated Maintainer maintainers = 9;
+  
+  // Available package opaque data
+  //
+  // A plugin can define further key-values for data which does not (yet) fit
+  // with the core.packaging.AvailablePackageDetail.
+  map<string, bytes> opaque_data = 10;
 }
 
 // PackageRepository
 //
 // A PackageRepository defines a repository of packages for installation.
 message PackageRepository {
```

Comment:
Hmm, we need this for references do we? Fine to leave this one if so... we can move it later as needed. Actually, there's a PackageRepositoryReference below, so perhaps not?

Reply:
Delete this one too pls.